models/solutions.js

```diff
@@ -64,6 +64,10 @@ module.exports = {
     rootOrganisations : {
       type : [String],
       default : []
+    },
+    link: {
+      type: String,
+      index: true
```

Comment:
@deepakr123 - Link should be unique

Reply:
@deepakr123 - Add index here for programId, programExternalId and solutionExternalId. solutionExternalId should be a unique index.

Reply:
resolved